pkg/cpu/counter_darwin.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Minio Cloud Storage, (C) 2019 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cpu
+
+import (
+	"errors"
+)
+
+func newCounter() (counter, error) {
+	return counter{}, errors.New("cpu metrics not implemented for darwin platform")
```

Comment:
We need to finalize the approach for darwin and windows platforms.

Reply:
Agreed. But, I don’t think that should be a blocker for this PR.

Reply:
IMO this should be documented.

Reply:
Will do